src/init.cpp

```diff
@@ -1258,6 +1259,14 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         }
     }
 
+    if (!args.IsArgSet("-cjdnsreachable")) {
+        SetReachable(NET_CJDNS, false);
+    }
+    // Now IsReachable(NET_CJDNS) is true if:
```

Comment:
In commit 3064e8b1474f955c2506711c57dfbf3b7af48686, how about the regular -reachable flag? What's the interaction here?

Reply:
By "regular -reachable flag" you mean `-onlynet=cjdns`? The interaction is that we can connect to CJDNS if `-cjdnsreachable` is given and CJDNS is not restricted by `-onlynet=`:
```cpp
    // 1. -cjdnsreachable is given and
    // 2.1. -onlynet is not given or
    // 2.2. -onlynet=cjdns is given
```

Reply:
Dismiss this comment, I was very confused.